### DISCUSSION TITLE
Few of my suggestions (Mostly Combat Related)

### DISCUSSION THREAD

**Discussion by ijaFree** [Stardust-Labs-MC/StardustMC#20] [Ideas]
Small/Simple Suggestions:
Increase the sell value of Enchanted Golden Apples
+ Designated warp for a PvP Arena/Area
+ Orb of Ankou (GM4)
+ In-game skip Weather vote system

Combat/Difficulty Ideas:
I noticed that a lot of the negative feedback upon the initial Sudden Death addition were the features that could not be toggleable. (Slowness when taking fall damage, Mob abilities, etc.)
I think that one of the best ways for players to pick and choose their difficulty is to have a designated location where players could go to for a challenge. Some sort of dungeon or other specific areas of the world could host tougher adversaries for those seeking out a challenge to explore and conquer. Those who dislike combat or anything tougher than what is already present could simply avoid the area.

My personal pitch for something like this would be a tower/floor based dungeon where the difficulty scales the higher you climb the tower.

The tower itself could be located near spawn but the "inside" of the tower/tower floors could be rooms that you could teleport to from the main tower to prevent lag and to give more flexibility to the uniqueness of each floor. (Ex: Floor 1 could be just a 32x32 normal box arena but Floor 2 could be an entirely underwater room, Floor 3 could be in the nether with Piglin Brutes and Floor 5 could be a Raid boss room deep underground)

As for how you could obtain loot/rewards from the Star Tower (the name ill be giving it to make it easier to read), You could have different types of loot systems and different types of new custom loot

First Time Only Completion Rewards (To entice players to try it out)
Ex: Upon first clearing Floor 1, you get X amount of stars/xp
Ex: Upon first clearing Floor 5, you get a custom X item
Not every floor has to have a First Time Completion Reward.

Custom RNG Mob Drop Rewards (To encourage re-playability)
Ex: Floor X has X amount of mobs and each mob has an X chance to drop a raw gold ingots
Ex: Floor X is a Boss Floor and the boss has 1 guaranteed drop and a 10% chance to drop a special drop

Completion Rewards (After FTO Completion Rewards have already been claimed)
Ex: Every time you clear Floor 1 after your first time clearing it, you are guaranteed 3 diamonds.
Not every floor has to have a Completion Reward.

As for the difficulty, loot and custom items, I would recommend items that don't blatantly disregard vanilla combat mechanics and scaling like chestplates with 10 armor and 4 armor toughness, swords with 30 atk damage, boots with +0.3 speed, etc. I also wouldn't make any of the replayable content OP like a floor where the mobs have a 10% chance to drop a diamond and every time you complete that floor, you'd get 100 stars. Basically, nothing blatantly overpowered to the point where everyone would feel forced to climb the Star Tower.

Here are a few random items I made as examples of custom items:
![2023-08-22_01 30 20](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/9aa10402-ab6f-445e-a3c5-347b06386e53)
![2023-08-22_01 43 42](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/1333eca2-c691-4fbd-931d-7f1c1c1aa25f)
![2023-08-22_01 51 23](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/da4b1770-6acf-4066-8672-42c2df4bc1ae)
![2023-08-22_01 56 54](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/1dc94067-cc76-4b57-b8cb-58a907ce3d25)
![2023-08-22_02 01 40](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/59ed6209-76e7-4ff4-a5bf-9e814c66540a)
Me wearing some silly gear:
![2023-08-22_02 03 37](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/c442135f-239c-43a4-ab15-3b2f047b50ac)

Before I go into the mobs themselves, id like to note that the maximum vanilla Armor and Armor toughness value is 30 Armor and 20 Toughness and the maximum value of protection from the protection enchant is 20 (Protection V on 4 pieces). Netherite is 20 Armor and 12 toughness. For a server like this, going above Netherite's armor values for custom gear should be done with absolute caution and care for current balance and should only be done so if the equipment has drawbacks to compensate for the armor or the armor is VERY hard to obtain. I greatly advise AGAINST modifying/raising Minecraft's vanilla Armor/Protection cap as Full Protection Netherite itself is already very hard to kill with Vanilla means and would be a balancing nightmare.

That aside, here are some custom mobs I made.
![2023-08-22_00 58 13](https://github.com/Stardust-Labs-MC/StardustMC/assets/142778929/daa75271-c747-40fa-876a-68513547f0b3)
These are simple mobs with custom items. Basically just heavily geared skeletons for now but with plugins, they could have special abilities.

     Types of Floors:
Basic Floors: Room with a few mobs and potential change of terrain. Kill all the mobs to complete the floor
Ex: Basic Box Room with 6 Fully Equipped Wither Skeletons or a Fully Submerged Room with Modified Guardians.

Survival Floors: Room which spawns waves of mobs. Survive either a specific number of waves to clear the floor. Terrain can vary.

Boss Floors: Room with a Boss. Terrain can vary.

     Star Tower Rules/Properties:
Keep Inventory is OFF. If you die in the tower, instead of having to climb it again to reach your loot or paying to directly TP into a dangerous room where you died, you could pay an NPC at the Tower to retrieve your loot for you.

You can leave the tower floors only after you clear them. Otherwise, you'll have to die and pay for your loot.

Every time you clear a floor, you'll have access to an Ender Chest to store loot or bring some extra resources to help continue your climb.

While in the tower, all forms of teleportation are disabled. You cannot set homes within the tower.

You can form parties of 3 players max. If one party member dies within the Tower, they have a 25 second timer to be "revived" in the tower. To revive a party member, one of the members must stand where the other party member died for 5 seconds.

Placing blocks/buckets will not be allowed in the tower.

You can not select which floor you want to start at. Every run must start at Floor 1.

Side Note:
I understand how much work would be needed to add such a concept. Especially the stuff that can't already be done with command blocks like a party system or custom mob/item abilities. The time it would take to design and build. But if interested, I'm definitely open to help test, design and work out these concepts to something unique that can potentially be added to the server. I've made and tested content like this for some close friends of mine and we had lots of fun with it. It'd be amazing if I could share the concept with you guys as well! I'm also very open to discussing other combat related ideas that we could implement as I do strongly believe that Minecraft's current problem is its lack of content regarding combat and the staleness of the late-game.

**Comment by ijaFree** [comment]
PS: 
This is the first time I've ever used github or posted a suggestion like this. Please give me tips/advice on how I can not be a complete noob lol thx

**Reply by aisukuma** [reply]
Looks pretty good! I would probably update the post with some better markdown to indicate categories/sections. You can view Github's markdown system [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
